tests/sentry/tasks/test_relay.py

```diff
@@ -130,6 +148,7 @@ def test_invalidate(
     task_runner,
     entire_organization,
     redis_cache,
```

Comment:
it should be possible to remove every usage of the `redis_cache` fixture now. The default cache is now redis.